### PR TITLE
torq: init at 0.16.9

### PR DESCRIPTION
**Pull request by mmilata**
###### Description of changes

https://github.com/lncapital/torq - capital management tool for Lightning Network nodes

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/blockchains/torq/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```
Comment (prusnak) [inline pkgs/applications/blockchains/torq/default.nix:1]:
```suggestion
{ lib
, buildGoModule
, fetchFromGitHub }:

```

File: pkgs/applications/blockchains/torq/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/lncapital/torq/build.Repository=github.com/${src.owner}/${src.repo}"
+    "-X github.com/lncapital/torq/build.overrideBuildVer=${version}"
+  ];
+
+  meta = with lib; {
+    description = "capital management tool for lightning network nodes";
```
Comment (prusnak) [inline pkgs/applications/blockchains/torq/default.nix:25]:
```suggestion
    description = "Capital management tool for lightning network nodes";
```

File: pkgs/applications/blockchains/torq/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    license = licenses.mit;
+    homepage = "https://github.com/lncapital/torq";
+    maintainers = with maintainers; [ mmilata prusnak ];
+    platforms = platforms.all;
```
Comment (prusnak) [inline pkgs/applications/blockchains/torq/default.nix:29]: This is not required and it's better to omit it completely (`go.meta.platforms` will be used which is what you want most of the times).

```suggestion
```

**Review by prusnak — CHANGES_REQUESTED** [review]
Left couple of nitpicks, otherwise looks OK

**Comment by 2Fast2BCn** [comment]
Starting from Torq v0.16.15:
To get our version number in our distributed packages we migrated our version tracking to `go generate build/version.go`
I hope this doesn't cause problems for this integration.
If this integration uses docker images then our version should already be in that image.
If the integration builds Torq then please add `go generate build/version.go` before building.

**Pull request merged**